x-pack/filebeat/module/zeek/README.md

```diff
@@ -0,0 +1,73 @@
+# Zeek (Bro) module
+
+## Caveats
+
+* Module is to be considered _alpha_.
+* Field names will be changing for 7.0 to comply with Elastic Common Schema (ECS).
```

Comment:
I think we'll only include this module in 7.0 anyway.

Reply:
Did not pay attention to this sentence. Will remove this.